### PR TITLE
Fixes gitignore and dependency issue

### PR DESCRIPTION
**Pull request by nimitzshah**

### CODE REVIEW

**Comment by nimitzshah** [comment]
@kylemac want me to bump version to 0.1.0?

**Comment by kylemac** [comment]
yeah, but we'll do that on master. you don't want to do that in the PR, as a habit, in case we want to incorporate other PRs/changes as part of the next release

File: readme.md [hunk 1]
```diff
@@ -14,3 +14,8 @@ A template for roots v3 configured with marionette
 - **github_username** (name of github user)
 - **description** (a short description of the template)
 - **models** (optional, space-separated list of models to generate templates for)
+
+### Amazing links
```
Comment (kylemac) [inline readme.md]: change to. I'm not sure I'd give them "amazing"
```markdown
### Marionette Resources
```

**Comment by kylemac** [comment]
1 comment then :+1: to merge

**Pull request merged**